NEWS.md

```diff
@@ -3,7 +3,7 @@ Julia v1.7 Release Notes
 
 New language features
 ---------------------
-
+* count and findall now accept AbstractChar as first argument to search for characters in strings ([#38675](https://github.com/JuliaLang/julia/pull/38675)).
```

Comment:
```suggestion
* `count` and `findall` now accept an `AbstractChar` argument to search for characters in strings ([#38675]).
```
(Note that the issue links are automatically generated, so you shouldn't put the URL in manually.)